NetCoreTeamCity/Models/TestOccurrence.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace NetCoreTeamCity.Models
+{
+    public class TestOccurrence
+    {
+        public string Id { get; set; }
+        public string Href { get; set; }
+        public string Name { get; set; }
+        public TestOccurrenceStatus Status { get; set; }
+        public bool? Ignored { get; set; }
+        public int? Duration { get; set; }
+        public int? RunOrder { get; set; }
+        public bool? Muted { get; set; }
+        public bool? CurrentlyMuted { get; set; }
+        public bool? currentlyInvestigated { get; set; }
+        public Test Test { get; set; }
+        public Build Build { get; set; }
```

Comment:
This should be BuildModel type if its a part of requests to API, or Build if its a model used by API user - please see BuildModelToBuildConventer